src/net.cpp

```diff
@@ -2102,11 +2102,18 @@ void CConnman::ThreadOpenMasternodeConnections()
 
     auto& chainParams = Params();
 
+    bool didConnect = false;
     while (!interruptNet)
     {
-        if (!interruptNet.sleep_for(std::chrono::milliseconds(1000)))
+        int sleepTime = 1000;
+        if (didConnect) {
+            sleepTime = 100;
+        }
+        if (!interruptNet.sleep_for(std::chrono::milliseconds(sleepTime)))
```

Comment:
same line or brackets

Reply:
Not in the scope of this PR

Reply:
Not really out of scope since you changed `1000` to `sleepTime`, but this is non-blocking.